sdk/eventhub/azure-eventhubs/azure/eventhub/_eventprocessor/ownership_manager.py

```diff
@@ -54,8 +55,26 @@ def claim_ownership(self):
             self.fully_qualified_namespace, self.eventhub_name, self.consumer_group
         )
         to_claim = self._balance_ownership(ownership_list, self.cached_parition_ids)
-        claimed_list = self.checkpoint_store.claim_ownership(to_claim) if to_claim else []
-        return [x["partition_id"] for x in claimed_list]
+        self.owned_patitions = self.checkpoint_store.claim_ownership(to_claim) if to_claim else []
```

Comment:
nit: patitions->partitions.

And re: our OOB discussion earlier, I think this should be safe to use in a shared context since we're replacing the owned_partitions instead of mutating it, assuming that the blob and container clients are also safe. (down through the checkpoint_store)

Reply:
Wow... consistent spelling fail.... that's not a "nit" >_<

Thanks for investigating this :)